docs/tutorials/configure/scuttlebot.md

```diff
@@ -16,138 +16,219 @@ languages: [ "python", "go" ]
-You'll use this panel to set the motor's `power` level.
+**Left encoder:**
 
-![Power level adjustment](/tutorials/scuttlebot/pi-moverhmotor.png)
+Navigate to the **Create component** menu.
+Enter `lenc` as the **name** for your encoder, select the type `encoder`, and select the `AMS-AS5048` model.
 
-Be careful when activating your robot!
-Start with the power level set to 10% and increase it incrementally until the wheel rotates at a reasonable speed, clicking **Run** at each increment.
+Click **Create component**.
 
-{{% alert title="Tip" color="tip" %}}
-A "whining" sound emitted from the motor indicates that the power level is not high enough to turn the armature.
-If this happens, increase the power level by 10% increments until it starts to turn.
-{{% /alert %}}
+![Configuration of an AS5048 encoder in the Viam app config builder.](/components/encoder/configure-am5.png)
 
-At this point, the right-side wheel should be working.
+Paste the following in the **Attributes** field:
 
-### Adding the left motor
+```json
+{
+    "board": "local",
+    "connection_type": "i2c",
+    "i2c_attributes": {
+        "i2c_bus": "main",
+        "i2c_addr": 64
+    }
+}
+```
+
+**Right encoder:**
+
+Navigate to the **Create component** menu.
+Enter `renc` as the **name** for your encoder, select the type `encoder`, and select the `AMS-AS5048` model.
 
-Now, you're ready to add the left-side motor.
-This will be similar to adding the right motor.
+Click **Create component**.
+
+Paste the following in the **Attributes** field:
+
+```json
+{
+    "board": "local",
+    "connection_type": "i2c",
+    "i2c_attributes": {
+        "i2c_bus": "main",
+        "i2c_addr": 65
+    }
+}
+```
 
-1. Name the component "left".
-2. Select "motor" from the **Type** drop-down.
-3. Select "gpio" from the **Model** drop-down.
-4. Click **Create Component**.
-5. Select `local` from the **Board** drop-down.
-6. Set the **Max RPM** attribute to `100`.
-7. Configure the motor's pins:
-   1. Switch the Component Pin Assignment Type to **In1/In2**.
-   2. Set **A/In1** to `12`.
-   3. Set **B/In2** to `11`.
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
 
-{{% expand "Click to view the raw JSON for the left motor" %}}
+Add the following JSON objects to the `components` array:
 
 ```json {class="line-numbers linkable-line-numbers"}
+{
+    "name": "lenc",
+    "type": "encoder",
+    "model" : "AMS-AS5048",
+    "attributes": {
+      "board": "local",
+      "connection_type": "i2c",
+      "i2c_attributes": {
+        "i2c_bus": "main",
+        "i2c_addr": 64
+      }
+    }
+},
+{
+    "name": "renc",
+    "type": "encoder",
+    "model" : "AMS-AS5048",
+    "attributes": {
+      "board": "local",
+      "connection_type": "i2c",
+      "i2c_attributes": {
+        "i2c_bus": "main",
+        "i2c_addr": 65
+      }
+    }
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Configure the motors
+
+The next step is to add the motors and make them spin their wheels.
```

Comment:
```suggestion
The next step is to add the motors and make them spin the wheels.
```
I think this is better because it doesn't sound as much like the idiom "spin your wheels." I got caught on this sentence.